src/PlaywightSharp.ProtocolTypesGenerator/Chromium/ChromiumProtocolTypesGenerator.cs

```diff
@@ -26,11 +26,6 @@ public async Task GenerateTypesAsync(RevisionInfo revision)
                 Directory.CreateDirectory(directory);
             }
 
-            if (revision.Local && File.Exists(output))

```

Comment:
If it's local great. If we already have the file, we wouldn't care, we have git to protect ourselves.